File: typescript/basic_grammar/operation/assignment_operator/logicalConjunction.ts
export const logicalConjunctionAssignmentOperator = () => {
  let a = 50
  a &= 21
  return a
}

export const decimalLogicalConjunctionAssignmentOperator = () => {
  let a = 50
  a &= 21.9
  return a
}

export const negativeLogicalConjunctionAssignmentOperator = () => {
  let a = 50
  a &= -21
  return a
}

console.log(logicalConjunctionAssignmentOperator())
console.log(decimalLogicalConjunctionAssignmentOperator())
console.log(negativeLogicalConjunctionAssignmentOperator())

File: python/basic_grammar/operation/assignment_operator/logical_conjunction.py
def logical_conjunction_assignment_operator():
    a = 50
    a &= 21
    return a

# 以下のコードは実行できない
# def decimal_logical_conjunction_assignment_operator():
#     a = 50
#     a &= 21.9
#     return a


def negative_logical_conjunction_assignment_operator():
    a = 50
    a &= -21
    return a

print(logical_conjunction_assignment_operator())
# print(decimal_logical_conjunction_assignment_operator())
print(negative_logical_conjunction_assignment_operator())